### ISSUE TITLE
Request: PyTorch update to 0.3.0

### ISSUE THREAD

**Issue by khanrc** [Kapeli/Dash-User-Contributions#1691]
docs: http://pytorch.org/docs/0.3.0/

**Comment by Kapeli** [comment]
// @iamaziz

**Comment by iamaziz** [comment]
Thanks for pointing this out. 

At the moment, I'm having an issue with `make`  on my machine. 
However, the steps to build and generate the updated docset are pretty simple:

```bash
git clone https://github.com/pytorch/pytorch.git
cd pytorch/docs
sphinx-build -M html source build
doc2dash --name PyTorch --icon source/_static/img/pytorch-logo-flame.png --enable-js --online-redirect-url http://pytorch.org/docs/ --force build/html
```

I'll try this again later at some point.

**Comment by reuben** [comment]
Fixed by #1717.

**Comment by khanrc** [comment]
thx :)
